### PR TITLE
Add maintainers

### LINKED ISSUE #49: Maintainers file
We should have maintainers

### PR DESCRIPTION
**Pull request by cnunciato**
Fixes #49

### CODE REVIEW

**Comment by cnunciato** [comment]
I've boldly proposed myself as project lead here mainly because it's still pretty nascent and starting to get cross-functional adoption, so we need someone who'll be able to review submissions in context, answer questions, give guidance and make sure the library continues to evolve in a way that keeps everyone happy.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+* [Chris Nunciato](http://github.com/cnunciato)
+
+### Lieutenants
+
+* [Nathan Smith](https://github.com/smith)
+* [Andy Paroff](https://github.com/unikitteh)
+
+### Maintainers
+
+* [Travor Bramble](https://github.com/TrevorBramble)
```
Comment (TrevorBramble) [inline MAINTAINERS.md]: Spelling!

Comment (cnunciato) [inline MAINTAINERS.md]: Yikes, sorry. Stupid fingers.

**Comment by TrevorBramble** [comment]
:+1: Looks good to me.

**Pull request merged**